docs/articles/documentation/reference/test-api/clientfunction/with.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title: ClientFunction.with Method
+permalink: /documentation/reference/test-api/clientfunction/with.html
+---
+# ClientFunction.with Method
+
+Overwrites [client function options](../global/clientfunction.md#options).
+
+```text
+clientFunction.with( options ) → ClientFunction
+```
+
+`with` returns a new client function with a different set of options that includes options
+from the original function and new `options` that overwrite the original ones.
```

Comment:
Can we say:
`with` returns a new client function with a different set of options. This includes options from the original function and new `options` that overwrite the original.